java-checks/src/main/java/org/sonar/java/checks/StaticMultithreadedUnsafeFieldsCheck.java

```diff
@@ -61,8 +68,20 @@ public void visitNode(Tree tree) {
   }
 
   private static boolean isForbiddenType(VariableTree variableTree) {
```

Comment:
The method should probably be renamed since it's no longer just about the type. Maybe just `isForbidden`.